lib/Robo/Traits/CliRunnerTrait.php

```diff
@@ -65,13 +65,27 @@ protected function bootstrap()
          * since they have side effects and can make other
          * Robo Tasks fail (i.e. failed database connection).
          */
-
         $root = __DIR__ . '/../../../';
 
         require $root . 'config.php';
         require $root . 'config_override.php';
         require_once $root . 'include/entryPoint.php';
 
+        // Load up the config.test.php file. This is used to define configuration values for the test environment.
+        $testConfig = [];
+
+        if (is_file($root . 'tests/config.test.php')) {
```

Comment:
it occurs to me now, wouldn't this effect all the Robo tasks that use CliRunnerTrait?

Reply:
Also: when I run `./vendor/bin/robo tests:unit`, at least on the StateChecker-removal branch, the `config.php` file is overwritten by the contents of the `config.test.php`.